lib/webdriver-plus.ts

```diff
@@ -201,11 +242,11 @@ Object.assign(WebElement.prototype, {
     return new WebElementRect(await this.getRect());
   },
   mouseMove(this: WebElement, params: {x?: number, y?: number} = {}): WebElementPromise {
-    // Unfortunately selenium-webdriver typings at this point (Nov'18) are one major version behind,
-    // and actions are incorrect.
-    // {bridge: true} allows support for legacy actions, currently needed for Chrome (Jan'19).
-    const actions = (this.getDriver() as any).actions({bridge: true});
-    const p = actions.move({origin: this, ...params}).perform();
+    const p = this.getDriver().withActions((actions) => actions.move({origin: this, ...params}));
     return new WebElementPromise(this.getDriver(), p.then(() => this));
   },
+  async hasFocus(this: WebElement): Promise<boolean> {
+    const active = this.getDriver().switchTo().activeElement();
+    return this.getDriver().executeScript((a: any, b: any) => (a === b), this, active) as Promise<boolean>;
```

Comment:
Would it be simpler to compare WebElement ids in webdriver than `executeScript` or is it less reliable?

Reply:
That works too. I took current approach from WebElement.equal method of webdriver, but reimplemented because that method doesn't actually work right. I can't tell when that's better though, so changing to comparing IDs directly.